README.md

```diff
@@ -183,10 +183,28 @@ QROM.
 Mix network key management and distribution is handled by the
 directory authority system, a decentralized voting protocol that can
 tolerate (1/2 * n)-1 node outages.  Clients and mix nodes can talk to
-the dirauth system to get a published PKI document which is
+the dirauth (directory authority) system to get a published PKI document which is
 essentially a view of the network which contains public cryptographic
 keys and network connection information.
 
+The mix descriptors are signed by the mix nodes. Each dirauth also signs their
+interactions in the voting protocol and the final published PKI document.
+
+Mix nodes and dirauth (directory authority) nodes use our hybrid post quantum signature
+scheme which is currently:
+
+* ed25519 + Sphincs+
+
+Note that we could have just used Sphincs+ since hash based signature schemes have a
+solid theoretical foundation, however there can still be mistakes in the hyper tree design
+and so pairing it with ed25519 is prudent defense in dept.
+
+Clients also use this signature scheme to verify PKI documents.
+
+Currently, we are using the *SHAKE-256f* parameterization of Sphincs+ however we will
+soon migrate away from that parameter set as it contains a NIST primitive we would
+like to avoid.
```

Comment:
We use some terminology here that isn't yet introduced, e.g. terms PKI and "mix descriptor" should be explained:
e.g.
"Clients and mix nodes can talk to ..." 
with
"Mix nodes publish their IP addresses and per-epoch keys in a self-signed "Mix Descriptor" to the directory authority nodes, which vote and produce a multisignature PKI document each Epoch. Clients receive this document each epoch and verify there is a threshold of signatures from the authorities.

Reply:
Is there a particular reason we want to avoid SHAKE-256f?

Reply:
> Is there a particular reason we want to avoid SHAKE-256f?

usually cypherpunks don't need to explain to other cypherpunks about avoiding NIST cryptographic primitives, yet that's what i'm doing here. we should not want to go to bed with NIST.